roles/switch_config/tasks/main.yml

```
@@ -1,7 +1,21 @@
 ---
 - name: Load switch configuration
   ansible.builtin.set_fact:
-    switch_vars: "{{ (lookup('vars', inventory_hostname, default='')) }}"
+    switch_vars: >-
+      {{
+        lookup('vars', inventory_hostname, default={}) |
+        default(
+            (
+              (
+                (
+                  hostvars[inventory_hostname]["cifmw_switch_config_scenarios"]
+                  [cifmw_switch_config_scenario_name]| default({})
```

care to improve spacing here?
```suggestion
                  [cifmw_switch_config_scenario_name] | default({})
```

---

done!